source/Octo.Tests/Commands/VersionTestFixture.cs

```diff
@@ -46,11 +46,18 @@ public void ShouldPrintCorrectVersionNumber()
 
         static string AssemblyPath()
         {
-            var codeBase = Assembly.GetExecutingAssembly().CodeBase;
-            var uri = new UriBuilder(codeBase);
-            var root = Uri.UnescapeDataString(uri.Path);
-            root = root.Replace('/', Path.DirectorySeparatorChar);
-            return root;
+            #if NETFRAMEWORK
```

Comment:
We use [this](https://github.com/OctopusDeploy/OctopusShared/blob/dfc6158202c34f7b4815b8e490390d12d91d7555/source/Octopus.Shared/Util/AssemblyExtensions.cs#L28-L36) code elsewhere, can't remember if this references Share or not!

Reply:
Nah, CLI doesn't reference shared.
All the old code that called `CodeBase` is deprecated / replaced by `.Location`.
Shared should get updated at some point.